gotime/go-time-101.md

```diff
@@ -370,7 +370,7 @@ I am working on a security issue that I'll try not to spoiler, because that woul
 
 **Johan Brandhorst:** Your call is important to us!
 
-**Roberto Clapis:** You're number 741 in line. Please wait. \[laughter\] \[unintelligible 00:47:18.26\]
+**Roberto Clapis:** You're number 741 in line. Please wait. \[laughter\] Let's hope not.
```

Comment:
Looks like this unintelligible was originally mis-attributed and should be listed as Johan. Can you fix that?